src/controllers/commands/CommandController.cpp

```diff
@@ -637,6 +638,37 @@ void CommandController::initialize(Settings &, Paths &paths)
             return "";
         });
 
+    this->registerCommand("/popout", [](const QStringList &words,
+                                        ChannelPtr channel) {
+        if (words.size() < 2)
+        {
+            if (!channel->isTwitchChannel() || channel->isEmpty())
+            {
+                channel->addMessage(makeSystemMessage(
+                    "Usage: /popout <channel>. You can also use the "
+                    "command without arguments in any twitch channel to "
+                    "open it's popout chat."));
```

Comment:
```suggestion
                    "command without arguments in any Twitch channel to "
                    "open its popout chat."));
```

Channeling my inner @Felanbird